environment-dev.yml

```diff
@@ -70,6 +70,8 @@ dependencies:
 - versioneer
 - python-wget
 - xarray
+- deepdiff
```

Comment:
We do dependency management in [these files](https://github.com/a-r-j/graphein/tree/master/.requirements)